tests/unit/src/test/scala/tests/RenameLspSuite.scala

```diff
@@ -391,6 +391,36 @@ object RenameLspSuite extends BaseLspSuite("rename") {
     breakingChange = (str: String) => str.replaceAll("Int", "String")
   )
 
+  renamed(
+    "macro-annotation",
+    """|/a/src/main/scala/a/Main.scala
+       |package a
+       |import io.circe.generic.JsonCodec
+       |trait LivingBeing
+       |@JsonCodec sealed trait <<An@@imal>> extends LivingBeing
```

Comment:
What happens if you rename an `Animal` reference from another file?

Reply:
That works - I can add another test for that.

Reply:
It would be good to add a test for this I think just in case.

Reply:
Added another case, fortunately works :sweat_smile: